daemon/main.go

```diff
@@ -405,20 +401,20 @@ func init() {
 		"Use a single cluster route instead of per node routes")
 	flags.StringVar(&socketPath,
 		"socket-path", defaults.SockPath, "Sets daemon's socket path to listen for connections")
-	flags.StringVar(&config.RunDir,
+	flags.StringVar(&option.GetConfig().RunDir,
 		"state-dir", defaults.RuntimePath, "Directory path to store runtime state")
-	flags.StringVarP(&config.Tunnel,
+	flags.StringVarP(&option.GetConfig().Tunnel,
 		"tunnel", "t", "vxlan", `Tunnel mode "vxlan" or "geneve"`)
 	flags.IntVar(&tracePayloadLen,
 		"trace-payloadlen", 128, "Length of payload to capture when tracing")
 	flags.Bool(
 		"version", false, "Print version information")
 	flags.Bool(
 		"pprof", false, "Enable serving the pprof debugging API")
-	flags.StringVarP(&config.DevicePreFilter,
+	flags.StringVarP(&option.GetConfig().DevicePreFilter,
 		"prefilter-device", "", "undefined", "Device facing external network for XDP prefiltering")
-	flags.StringVarP(&config.ModePreFilter,
-		"prefilter-mode", "", ModePreFilterNative, "Prefilter mode { "+ModePreFilterNative+" | "+ModePreFilterGeneric+" } (default: "+ModePreFilterNative+")")
+	flags.StringVarP(&option.GetConfig().ModePreFilter,
+		"prefilter-mode", "", option.ModePreFilterNative, "Prefilter mode { "+option.ModePreFilterNative+" | "+option.ModePreFilterGeneric+" } (default: "+option.ModePreFilterNative+")")
```

Comment:
These strings shouldn't have been changed, I think? they're the actual CLI arguments, right?